OrbitGl/EventTrack.cpp

```diff
@@ -76,8 +76,7 @@ void EventTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingM
 
   const Color kWhite(255, 255, 255, 255);
   const Color kGreenSelection(0, 255, 0, 255);
-  const CaptureData* capture_data = time_graph_->GetCaptureData();
-  CHECK(capture_data != nullptr);
+  const CaptureData& capture_data = app_->GetCaptureData();
```

Comment:
>Moving the capture data to the App level means that we can't have different timegraphs loaded at once. Concretely, this will duplicate the capture data in the introspection windows since introspection has its own timegraph that will access the "global" capture data from the App.

Okay, so if this is the case and we can't use app's capture data here, we should really pass the capture data all the way down to the tracks that use it. So EventTrack could have a pointer to the capture data to be used. 

WDYT?

Reply:
As discussed online, I agree